### ISSUE TITLE
Feature Request: a way to check for stale feeds.

### ISSUE THREAD

**Issue by VideoMangler** [brentsimmons/NetNewsWire#1107]
I didn't see this request yet, so here it goes: the "old" NetNewsWire had a way to check for stale feeds. If I remember correctly, there was a window in which you could show all feeds that had received no updates in a while, you could fill in the amount of days etc. It even showed those feeds in a brown color :) This would be -for me- the number one request. I'm surprised nobody brought it up yet (though I might not have searched thoroughly enough), as it can be tedious for people with hundred's of feeds to check the latest article dates by hand... Other than this, I think the current version is already very, very useable, thanks!

**Comment by brentsimmons** [comment]
We meant to get Dinosaurs in 5.0, but we decided to punt for a later release.

**Comment by VideoMangler** [comment]
Right, Dinosaurs is what it was called. I'm happy to know that it will be part of a later release, thanks!

**Comment by vincode-io** [comment]
Would we also want statistics for how long a feed takes to return? This might address #2476.

**Comment by stuartbreckenridge** [comment]
What do we define as _stale_? Repeated 404s over a specific time period?

**Comment by grischard** [comment]
Even if it’s a 200 code but there are no new posts for a certain amount of time, then the lights are on but nobody’s home.

**Comment by pimoore** [comment]
Are there plans to add this feature to the iOS/iPadOS versions as well?

**Comment by brentsimmons** [comment]
Dinosaurs feature is shipping in 7.1.
